src/dmd/mars.d

```diff
@@ -615,15 +615,16 @@ private int tryMain(size_t argc, const(char)** argv, ref Param params)
         removeHdrFilesAndFail(params, modules);
 
     // Scan for functions to inline
-    if (params.useInline)
+    foreach (m; modules)
     {
-        foreach (m; modules)
+        if (global.params.useInline || m.hasAlwaysInlines)
         {
-            if (params.verbose)
+            if (global.params.verbose)
```

Comment:
This is unnecessary, as is the addition of `global` above.

Reply:
```suggestion
            if (params.verbose)
```